adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/providers/ActilityEnterpriseProvider.java

```diff
@@ -0,0 +1,95 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.adapter.lora.providers;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import org.eclipse.hono.adapter.lora.GatewayInfo;
+import org.eclipse.hono.adapter.lora.LoraMetaData;
+import org.springframework.stereotype.Component;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * A LoRaWAN provider with API for Actility.
```

Comment:
this should probably be more specific now that we distinguish between different types of Actility ...